### PR TITLE
[Conan]Cmake install prefix

### PR DESCRIPTION
**Pull request by RAJAGOPALAN-GANGADHARAN**

### CODE REVIEW

File: Tools/qt/conanfile.py [hunk 1]
```diff
@@ -125,6 +125,9 @@ def build(self):
         else:
             ninja_flags = None
 
+        if "CMAKE_INSTALL_PREFIX" in cmake.definitions:
+            del cmake.definitions["CMAKE_INSTALL_PREFIX"]
```
Comment (annulen) [inline Tools/qt/conanfile.py]: Please remove this commit

File: Tools/qt/build-qtwebkit-conan.py [hunk 2]
```diff
@@ -159,6 +160,9 @@ def create_profile(compiler, arch):
 if args.build_type:
     os.environ["CMAKE_BUILD_TYPE"] = args.build_type
 
+if args.install_prefix:
+    os.environ["CMAKE_INSTALL_PREFIX"] = args.install_prefix
```
Comment (annulen) [inline Tools/qt/build-qtwebkit-conan.py:164]: `else` implementation is missing.

File: Tools/qt/conanfile.py (hunk 1 above)
Comment (RAJAGOPALAN-GANGADHARAN) [reply Tools/qt/conanfile.py]: Ok will do

File: Tools/qt/build-qtwebkit-conan.py (hunk 2 above)
Comment (RAJAGOPALAN-GANGADHARAN) [reply Tools/qt/build-qtwebkit-conan.py:164]: else it will be None by default

File: Tools/qt/conanfile.py [hunk 3]
```diff
@@ -125,8 +125,10 @@ def build(self):
         else:
             ninja_flags = None
 
-        if "CMAKE_INSTALL_PREFIX" in cmake.definitions:
-            del cmake.definitions["CMAKE_INSTALL_PREFIX"]
+        if "CMAKE_INSTALL_PREFIX" in os.environ:
+            cmake.definitions["CMAKE_INSTALL_PREFIX"] = os.environ["CMAKE_INSTALL_PREFIX"]
+        elif "QTDIR" in os.environ:
+            cmake.definitions["CMAKE_INSTALL_PREFIX"] = os.environ["QTDIR"]
```
Comment (annulen) [inline Tools/qt/conanfile.py:131]: QTDIR should be handled in build-qtwebkit-conan.py

Comment (RAJAGOPALAN-GANGADHARAN) [reply Tools/qt/conanfile.py:131]: You want me to make this check in build-qtwebkit-conan.py?

Comment (annulen) [reply Tools/qt/conanfile.py:131]: Yes

File: Tools/qt/build-qtwebkit-conan.py [hunk 4]
```diff
@@ -117,6 +117,7 @@ def create_profile(compiler, arch):
 parser.add_argument("--profile", help="Name of conan profile provided by user. Note: compiler and profile options are mutually exclusive", type=str)
 parser.add_argument("--arch", help="32 bit or 64 bit build, leave blank for autodetect", default="default", choices=['x86', 'x64'])
 parser.add_argument("--build_type", help="Name of CMake build configuration to use", default="Release", choices=['', 'Release', 'Debug'])
+parser.add_argument("--install_prefix", help="Cmake install prefix (defaults to Qt5 directory)", default=None)
```
Comment (annulen) [inline Tools/qt/build-qtwebkit-conan.py:120]: help="Set installation prefix to the given path (defaults to Qt directory)

**Pull request merged**